frontend/app/components/create-badge.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import Ember from 'ember';
+const EmberObject  = Ember;
+export default Component.extend({
+  didInsertElement() {
+    EmberObject.$(window).on('resize scroll', function(e) {
+      if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
+        this.document.getElementsByClassName('six wide column preview')[0].style.position = 'fixed';
```

Comment:
can we please avoid using vanilla javascript for ember components?
I've pointed it out previously as well. Can you make the changes please :smiley:

Reply:
Is this fine now @gabru-md ??